src/Telegram/Types/ChatMember.php

```diff
@@ -15,131 +22,27 @@
  */
 class ChatMember extends TelegramTypes
```

Comment:
This should be defined as an abstract method since a ChatMember will always be of a certain type.